CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 <!-- There should always be "Unreleased" section at the beginning. -->
 
 ## Unreleased
+- Allow setting an edismax as local parameters in query
+  - [**BC**] Add method `useEDisMaxGlobally` to `FulltextInterface`
+- Set `phraseBigramFields`, `boostQuery`, `phraseSlop` only when `eDisMax` is enabled
+- Mark `FulltextInterface::getDefaultQueryOperator` method as deprecated in favor of `FulltextInterface::getMinimumMatch()`
```

Comment:
Wouldn`t it be better to just drop the old method. Or you use this at VS?

Reply:
well it may be useful without an eDisMax enabled, so I'll keep it there and remove a `deprecated` notice